manifests/g/GetComposer/Composer/2.8.4/GetComposer.Composer.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GetComposer.Composer
+PackageVersion: 2.8.4
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
+ReleaseDate: 2024-12-11
+AppsAndFeaturesEntries:
+- DisplayName: Composer - PHP Dependency Manager
+  Publisher: getcomposer.org
+  ProductCode: '{7315AF68-E777-496A-A6A2-4763A98ED35A}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '{code:GetDefaultDir}'
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: PHP.PHP.8.4
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```

Might be an `x64` arch installer file?